ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
<p>More of a meta level question for discussion here...</p><p> </p><p>I wonder a little bit about the value of spelling out the individual examples here for all of these functions.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think there is value is keeping the individual examples the same level across the ncl_entries, just to keep the pages consistent

Reply:
Ok, maybe let's leave as is for now then and see how things scale.